roles/openshift_health_checker/openshift_checks/docker_storage.py

```diff
@@ -34,7 +34,7 @@ def run(self, tmp, task_vars):
             }
 
         # attempt to get the docker info hash from the API
-        info = self.execute_module("docker_info", {}, task_vars)
+        info = self.execute_module("docker_info", {}, task_vars=task_vars)
```

Comment:
In fact I'd like to internalize `task_vars` and `tmp` so we don't have to pass them around all the time.